modules/core/src/test/java/org/apache/synapse/deployers/LocalEntryDeployerTest.java

```diff
@@ -0,0 +1,120 @@
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Test class for LocalEntryDeployer
+ */
+public class LocalEntryDeployerTest {
+    /**
+     * Testing the deployment of a local entry
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testDeploy() throws Exception {
+        String inputXML = "<localEntry key=\"TestEntry\" xmlns=\"http://ws.apache.org/ns/synapse\">0.1</localEntry>";
+
+        OMElement inputElement = AXIOMUtil.stringToOM(inputXML);
+        LocalEntryDeployer localEntryDeployer = new LocalEntryDeployer();
+
+        SynapseConfiguration synapseConfiguration = new SynapseConfiguration();
+        AxisConfiguration axisConfiguration = synapseConfiguration.getAxisConfiguration();
+        ConfigurationContext cfgCtx = new ConfigurationContext(axisConfiguration);
+        SynapseEnvironment synapseEnvironment = new Axis2SynapseEnvironment(cfgCtx, synapseConfiguration);
+        axisConfiguration.addParameter(new Parameter(SynapseConstants.SYNAPSE_ENV, synapseEnvironment));
+        axisConfiguration.addParameter(new Parameter(SynapseConstants.SYNAPSE_CONFIG, synapseConfiguration));
+        cfgCtx.setAxisConfiguration(axisConfiguration);
+
+        localEntryDeployer.init(cfgCtx);
+        String response = localEntryDeployer.deploySynapseArtifact(inputElement, "sampleFile", null);
+
+        Assert.assertEquals("Local Entry not deployed!", "TestEntry", response);
+    }
+
+    /**
+     * Test updating a local entry
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testUpdate() throws Exception {
+        String inputXML = "<localEntry key=\"TestEntry\" xmlns=\"http://ws.apache.org/ns/synapse\">0.1</localEntry>";
+
+        OMElement inputElement = AXIOMUtil.stringToOM(inputXML);
+        LocalEntryDeployer localEntryDeployer = new LocalEntryDeployer();
+
+        SynapseConfiguration synapseConfiguration = new SynapseConfiguration();
+        AxisConfiguration axisConfiguration = synapseConfiguration.getAxisConfiguration();
+        ConfigurationContext cfgCtx = new ConfigurationContext(axisConfiguration);
+        SynapseEnvironment synapseEnvironment = new Axis2SynapseEnvironment(cfgCtx, synapseConfiguration);
+        axisConfiguration.addParameter(new Parameter(SynapseConstants.SYNAPSE_ENV, synapseEnvironment));
+        axisConfiguration.addParameter(new Parameter(SynapseConstants.SYNAPSE_CONFIG, synapseConfiguration));
+        cfgCtx.setAxisConfiguration(axisConfiguration);
+
+        localEntryDeployer.init(cfgCtx);
+        localEntryDeployer.deploySynapseArtifact(inputElement, "sampleFile", null);
+
+        String inputUpdateXML = "<localEntry key=\"TestEntryUpdated\" xmlns=\"http://ws.apache.org/ns/synapse\">0.1</localEntry>";
+
+        OMElement updatedElement = AXIOMUtil.stringToOM(inputUpdateXML);
+
+        String response = localEntryDeployer.updateSynapseArtifact(updatedElement, "sampleUpdateFile", "TestEntry", null);
```

Comment:
Don't have unnecessary line spaces when there is no any logic separation between code segments.

Reply:
Fixed with https://github.com/wso2/wso2-synapse/pull/1105